Changelogs for Digital Twin Manager UI and Microservice 1020.1.15 (#2380)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020.1.15-fixed-ui-issues-for-sample-asset-models.md

---
date: ""

> title: "Fixed UI issues for sample asset models"

Author: ```suggestion
title: "Improved naming and consistency in Asset model samples"
```

product_area: "Application enablement & solutions"
change_type:

>     - value: "change-VSkj2iV9m"
>       label: "Fix"

Author: ```suggestion
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
```

component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1512"
version: "1020.1.15"
---

> Updated the naming convention by removing "model" from the names of newly added sample asset models for better readability and uniformity. Also applied CSS to ensure descriptions in the sample asset model card are truncated with an ellipsis for consistency.

Author: ```suggestion
Removed "model" from the asset model sample names for simplicity. Improved the description and UI layout to maintain consistency.
```
